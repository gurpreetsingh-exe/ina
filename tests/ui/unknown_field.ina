// COMPILE_FAIL

type Foo = {
    a: &i32,
    b: i32
}

fn main() {
    let a = 34;
    let a = Foo { a: &a, c: 20 };
    let b = a.c;
}

