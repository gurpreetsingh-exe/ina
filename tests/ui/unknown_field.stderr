error: `Foo` has no field `c`
   --> unknown_field.ray:10:29
    |
 LL |     let a = Foo { a: &a, c: 20 };
    |                             ^^

error: `Foo` has no field `c`
   --> unknown_field.ray:11:13
    |
 LL |     let b = a.c;
    |             ^

